libros/packages/get_dewey_category.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python3
+
+import sys
+
+""" Gets the top level category from the dewey decimal system. 
+
+Usage:
+
+    python3 get_category.py category_number
+"""
+
+prime_categories = {
+    ('000', 'Computer Science, Information, and General Works', 'blue'),
+    ('100', 'Philosophy and Psychology', 'red'),
+    ('200', 'Religion', 'black'),
+    ('300', 'Social Sciences', 'purple'),
+    ('400', 'Language', 'green'),
+    ('500', 'Science', 'orange'),
+    ('600', 'Technology', 'pink'),
+    ('700', 'Arts and Recreation', 'yellow'),
+    ('800', 'Literature', 'brown'),
+    ('900', 'History and Geography', 'white'),
+}
+
+
+def get_category(dewey):
+    for category in prime_categories:
+        if (category[0][0] == dewey[0]):
```

Comment:
This checks the first character of the `000`, `100` etc value doesn't it?
Would it be better to use a `Dict[Tuple]` here instead of a `Set[Tuple]`?
You could key the Dict 1-9 then `return prime_categories.get(dewey[0])`

Also, are these categories likely to change much? 
Would it benefit from being stored in MongoDB?

Reply:
Good call, that makes sense.

No the list isn't likely to change at all. It's the master list of the Dewey decimal system, I don't think it's changed in decades.